mr_project/mr_project/settings.py

```diff
@@ -10,6 +10,7 @@
 
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/3.1/ref/settings/
+
```

Comment:
Any reason this is needed?

Reply:
I never touched settings.py - not sure.

Reply:
It's all good, just revert the change then